plugins/module_utils/modules.py

```diff
@@ -20,7 +20,7 @@
     UNIVERSAL_DDI_CLIENT_IMP_ERR = traceback.format_exc()
 
 
-class BloxoneAnsibleModule(AnsibleModule):
+class UniversalDdiAnsibleModule(AnsibleModule):
```

Comment:
```suggestion
class UniversalDDIAnsibleModule(AnsibleModule):
```

Quoting the [style guide](https://peps.python.org/pep-0008/) 
> Note: When using acronyms in CapWords, capitalize all the letters of the acronym. Thus HTTPServerError is better than HttpServerError.

Reply:
Fixed